sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_prebuilt_document_async.py

```diff
@@ -63,12 +63,10 @@ async def analyze_document():
             )
         result = await poller.result()
 
-    for idx, style in enumerate(result.styles):
-        print(
-            "Document contains {} content".format(
-                "handwritten" if style.is_handwritten else "no handwritten"
-            )
-        )
+    for style in result.styles:
+        if style.is_handwritten:
+            print("Document contains handwritten content: ")
```

Comment:
Do we not want to say that there was no handwritten content on the document? I dont really mind, just wondering because that means if people copy this they wont see the locations of the none handwritten content.

Reply:
Before it would print "Document contains handwritten content" for every instance of handwritten content. I think this is a good opportunity to show people how to use the spans to find the specific handwritten content (sans helper methods this beta). We could set a var to hold whether any handwritten content was found while iterating over the list and then print "No handwritten content" but since we aren't doing that for any of the other features like selection marks, entities, etc I left as-is. What do you mean by "locations of the none handwritten content"?

Reply:
I like your idea. I just meant also printing the spans for places where there's no handwritten content. But I think it's enough to show it with one, shouldnt be too hard for users to switch it up for whichever one they want. This LGTM as is then :)